rest/src/main/java/discord4j/rest/service/ApplicationService.java

```diff
@@ -119,4 +121,36 @@ public ApplicationService(Router router) {
             .exchange(getRouter())
             .bodyToMono(Void.class);
     }
+
+    public Flux<GuildApplicationCommandPermissionsData> getGuildApplicationCommandPermissions(long applicationId,
+                                                                                              long guildId) {
+        return Routes.GUILD_APPLICATION_COMMAND_PERMISSIONS_GET.newRequest(applicationId, guildId)
+                .exchange(getRouter())
+                .bodyToMono(GuildApplicationCommandPermissionsData[].class)
+                .flatMapMany(Flux::fromArray);
+    }
+
+    public Mono<GuildApplicationCommandPermissionsData> getApplicationCommandPermissions(long applicationId, long guildId,
+                                                                                         long commandId) {
+        return Routes.APPLICATION_COMMAND_PERMISSIONS_GET.newRequest(applicationId, guildId, commandId)
+                .exchange(getRouter())
+                .bodyToMono(GuildApplicationCommandPermissionsData.class);
+    }
+
+    public Mono<Void> modifyApplicationCommandPermissions(long applicationId, long guildId,
+                                                          long commandId,
+                                                          List<ApplicationCommandPermissionsData> permissions) {
+        return Routes.APPLICATION_COMMAND_PERMISSIONS_MODIFY.newRequest(applicationId, guildId, commandId)
+                .body(permissions)
```

Comment:
I think we need a wrapper around this field. The docs say the body is an object with a `permissions` field, not just the array.

Reply:
You're right, I have created a PR to implement the wrapper: https://github.com/Discord4J/discord-json/pull/75/files